### PR TITLE
reduced methods, removed PropCI package as infrequently used.  Added …

### PR DESCRIPTION
**Pull request by DrLynTaylor**
…more clarity to methods & made it clearer what's left to do

### CODE REVIEW

File: Comp/r-sas_ci_for_prop.qmd [hunk 1]
```diff
@@ -6,37 +6,37 @@ execute:
 
 ## Introduction
 
-There are many different methods of calculating a confidence interval for a proportion (percentage).
+There are several ways to calculate a confidence interval (CI) for a proportion. You need to select the method based on if you have a 1 sample proportion (1 proportion calculated from 1 group of subjects), or if you have 2 samples and you want a CI for the difference in the 2 proportions. The difference in proportion can come from either 2 independent samples (different subjects in each of the 2 groups), or can be matched (the same subject with 1 result in 1 group and 1 result in the other group \[paired data\]).
```
Comment (statasaurus) [inline Comp/r-sas_ci_for_prop.qmd:9]: Do you want the slashes here?

File: R/ci_for_prop.qmd [hunk 2]
```diff
@@ -187,33 +170,43 @@ proportion_ci_jeffreys(act2,conf.level=0.95) %>%
...
 
-### Blaker Method using PropCIs
+## Methods for Calculating Confidence Intervals for a matched pair proportion
 
-The blaker method is available in the the package PropCIs. In practice this method is rarely used.
+When you have 2 measurements on the same subject, the 2 sets of measures are not independent and you have matched pair of responses.
 
-```{r}
-blakerci(aeve, atot, conf.level=0.95, tolerance=1e-05)
+It is not usual to calculate the proportions separately for these responses, and then take a difference, instead other statistical methods such as Odds ratios, Risk Ratios and confidence intervals for those statistics are used.
```
Comment (statasaurus) [inline R/ci_for_prop.qmd:177]:
```suggestion
It is not usual to calculate the proportions separately for these responses, and then take a difference, instead other statistical methods such as odds ratios, risk ratios and confidence intervals for those statistics are used.
```

File: R/ci_for_prop.qmd [hunk 3]
```diff
@@ -187,33 +170,43 @@ proportion_ci_jeffreys(act2,conf.level=0.95) %>%
...
 
-```
+## Methods for Calculating Confidence Intervals for 2 independent samples proportion using cardx
```
Comment (statasaurus) [inline R/ci_for_prop.qmd:179]:
```suggestion
## Methods for Calculating Confidence Intervals for 2 independent samples proportion using {cardx}
```

**Review by statasaurus — APPROVED** [review]

File: Comp/r-sas_ci_for_prop.qmd (hunk 1 above)
Comment (DrLynTaylor) [reply Comp/r-sas_ci_for_prop.qmd:9]: no... issue with visual / source again :(

**Pull request merged**